bindings/electron/src/meths.rs

```diff
@@ -5194,6 +5194,15 @@ fn test_drop_testbed(mut cx: FunctionContext) -> JsResult<JsPromise> {
     Ok(promise)
 }
 
+// get_os
+fn get_os(mut cx: FunctionContext) -> JsResult<JsPromise> {
+    let ret = libparsec::get_os();
+    let js_ret = JsString::try_new(&mut cx, ret).or_throw(&mut cx)?;
+    let (deferred, promise) = cx.promise();
+    deferred.resolve(&mut cx, js_ret);
+    Ok(promise)
+}
```

Comment:
@touilleMan Generating a promise for that seems a little overkill

Reply:
I'd rather avoid adding sync method support in the bindings in order to avoid the added complexity


Most methods in the API are async anyway, and the remaining ones should work fine in async (given everything is async in javascript, there is not colored function issue there)